### ISSUE TITLE
Missing content in German version of consent form

### ISSUE THREAD

**Issue by MarvinMaechler** [datalad/open-brain-consent#8]
The following content does not appear in the German version: 

"However, by using additional data linked to your name (for example brain scans obtained from your medical records) one could potentially associate your imaging or other information in our database back to you. In addition a security breach (break in or cyber attack) might lead to someone being able to link you to your data. This risk is very low because your data are stored in a secure database, and the information about your identity is stored separately from the data themselves, linked only through a code.

We will keep the private portion (name, contact information etc.) of your data in a secure location for at least <x> years. This way if one of the researchers that obtained the data from us will find something in your brain scans that would have a diagnostic value we will be able to contact you. After this period of time we will destroy this information to protect your privacy."

Was that done intentionally?

**Comment by chrisfilo** [comment]
Yes. Those paragraphs come from the "Two access types version" and only "Single access type version" was translated to German.

**Comment by yarikoptic** [comment]
nah.... according to the paragraph count some paragraphs are indeed missing for the first version. And two paragraphs (2 and 3) collapsed into a single one.  @bpoldrack will fix everything! ;)
